src/solvers/refinement/string_constraint_generator_float.cpp

```diff
@@ -422,10 +422,10 @@ std::pair<exprt, string_constraintst> add_axioms_from_float_scientific_notation(
     int_type);
   array_exprt conversion_factor_table(
     array_typet(float_type, conversion_factor_table_size));
-  for(const auto &f : two_power_e_over_ten_power_d_table_negatives)
-    conversion_factor_table.copy_to_operands(constant_float(f, float_spec));
-  for(const auto &f : two_power_e_over_ten_power_d_table)
-    conversion_factor_table.copy_to_operands(constant_float(f, float_spec));
+  for(const auto &neg : two_power_e_over_ten_power_d_table_negatives)
```

Comment:
:pick: Don't use abbreviations `negative`, `position` - though obviously a massive improvement on what was there before. 
:pick: Rename the param `f` to `float_expr`?

Reply:
Done, and a separate commit doing the renaming of `f` is being prepared.